data/campaigns/tutorial03_seafaring.wmf/scripting/helper_functions.lua

```diff
@@ -80,7 +80,7 @@ end
 function get_waterway_fields(area)
    local fields = {}
    for i,field in pairs(area) do
-      if field.immovable and field.immovable.descr.type_name == "waterway" then
+      if field.immovable and field.immovable.descr.type_name == "waterway" then -- NOCOM crash
```

Comment:
Error is raised in `LuaMapObject::get` and is possibly related to https://github.com/widelands/widelands/issues/4259 and not caused by this branch.

Reply:
Does this still happen now?

Reply:
I have filed a new issue for this so it won't block this branch. The bug is already in master https://github.com/widelands/widelands/issues/4347